generatedTypes/incubator/TextField/index.d.ts

```diff
@@ -8,7 +8,7 @@ import { LabelProps } from './Label';
 import { FieldStateProps } from './withFieldState';
 import { FloatingPlaceholderProps } from './FloatingPlaceholder';
 import { CharCounterProps } from './CharCounter';
-interface TextFieldProps extends InputProps, LabelProps, FloatingPlaceholderProps, FieldStateProps, ValidationMessageProps, CharCounterProps {
+interface TextFieldProps extends InputProps, LabelProps, FloatingPlaceholderProps, FieldStateProps, ValidationMessageProps, Omit<CharCounterProps, 'maxLength'> {
```

Comment:
I'm not too familiar with this usage, why do you need to omit 'maxLength'? Because it exists on both TextField and CharCounter components? Or because in one it's required and in the other is optional?

Reply:
because in one it's required and in the other is optional?

Reply:
I see. So let's leave it optional then. It's not worth it.